Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -15,57 +15,78 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Octane_Repository;
+using ALM_Common.Data_Contracts;
+using Octane_Repository.BLL;
+using OctaneSDK.Entities.Base;
+using OctaneSDK.Entities.WorkItems;
+using OctaneSDK.Entities.Tests;
+using OctaneSDK.Entities.Requirements;
 
 namespace GingerCore.ALM
 {
     public class OctaneCore : ALMCore
     {
         public override ObservableList<ActivitiesGroup> GingerActivitiesGroupsRepo { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public override ObservableList<Activity> GingerActivitiesRepo { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public ProjectArea ProjectArea { get; private set; }
 
         public RestConnector mOctaneRestConnector;
         public EntityService entityService;
         protected LwssoAuthenticationStrategy lwssoAuthenticationStrategy;
         protected WorkspaceContext workspaceContext;
         protected SharedSpaceContext sharedSpaceContext;
+        protected OctaneRepository octaneRepository;
 
         public override bool ConnectALMProject()
         {
-            throw new NotImplementedException();
+            return this.ConnectALMServer();
         }
 
         public override bool ConnectALMServer()
         {
             try
             {
+                if (octaneRepository == null)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6235433)